cityofaustin/janis
# 4423 Spanish translation for "0" search results page

Zenhub Issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4423

Deployed:  https://janis-v3-es-no-results.netlify.com/


# Description

This PR will add the spanish translation to the "0 results" found within the search page. Also, The way language toggling was updating the page wasn't complete with the toggle and that is updated here as well. 

# Types of changes

# How can this be tested?
- From the deployed [link](https://janis-v3-es-no-results.netlify.com/)
- toggle language to Spanish, 
- hit search 
- [ ] search is in Spanish 
- search for something ('police')
- [ ] result(s) are in Spanish
- toggle english 
- [ ] results translate AND show different results (because the results are different in English)
- [ ] mess around more and see if it works well. 

<img width="1186" alt="Screen Shot 2020-06-09 at 7 28 19 PM" src="https://user-images.githubusercontent.com/15821138/84214209-5d8cbc80-aa88-11ea-8340-71288184daa5.png">


# Checklist:
